app/views/catalog/_uv.html.erb

```diff
@@ -1,8 +1,13 @@
 <iframe
     class="universal-viewer-iframe"
+    <% if Rails.env == 'test' %>
```

Comment:
Maybe it won't work to stub the entire template because that's what you are testing, but instead pull the `src` value into a helper method and then stub out that helper method for testing.

Let me know if that makes sense.